src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -0,0 +1,135 @@
+      expect(nextCol).to.eql(0);
+    });
+
+    it('should go to one row down cell', () => {
+      evt.key = 'ArrowDown';
+      rowAndColumnCount.rowCount = 2;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(1);
+      expect(nextCol).to.eql(0);
+    });
+
+    it('should go to one row up cell', () => {
+      evt.key = 'ArrowUp';
+      rowAndColumnCount.rowCount = 2;
+      rowIndex = 1;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(0);
+      expect(nextCol).to.eql(0);
+    });
+
+    it('should go to one column left cell', () => {
+      evt.key = 'ArrowLeft';
+      rowAndColumnCount.columnCount = 2;
+      colIndex = 1;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(0);
+      expect(nextCol).to.eql(0);
+    });
+
+    it('should go to one column right cell', () => {
+      evt.key = 'ArrowRight';
+      rowAndColumnCount.columnCount = 2;
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      expect(nextRow).to.eql(0);
+      expect(nextCol).to.eql(1);
+    });
+  });
+
+  describe('getRowAndColumnCount', () => {
+    let resolvedRowCount;
+    let resolvedColumnCount;
+    let resolvedRowElements;
+    const rootElement = {};
+
+    it('should return a rowElements, rowCount, and columnCount', () => {
+      resolvedRowCount = 1;
+      resolvedColumnCount = 1;
+      resolvedRowElements = [{}];
+      rootElement.getElementsByClassName = () => resolvedRowElements;
+      const { rowElements, rowCount, columnCount } = getRowAndColumnCount(rootElement);
+      expect(rowCount).to.equal(resolvedRowCount);
+      expect(columnCount).to.equal(resolvedColumnCount);
+      expect(rowElements).to.equal(resolvedRowElements);
+    });
+  });
+
+  describe('handleKeyPress', () => {
+    const sandbox = sinon.createSandbox();
+    let rowIndex;
+    let colIndex;
+    let evt;
+    const rootElement = {};
+
+    beforeEach(() => {
+      rowIndex = 0;
+      colIndex = 0;
+      evt = {
+        key: 'ArrowDown',
+        stopPropagation: sinon.spy(),
+        preventDefault: sinon.spy(),
+      };
+      evt.target = {
+        blur: sinon.spy(),
+        setAttribute: sinon.spy(),
+      };
+      rootElement.getElementsByClassName = () => [
+        { getElementsByClassName: () => [{ focus: () => {}, setAttribute: () => {} }] },
+      ];
+      sandbox.replace(handleKeyPress, 'default', sandbox.spy());
+    });
+
+    afterEach(() => {
+      sandbox.verifyAndRestore();
+    });
+
+    it('should stay current focus and focus the next cell', () => {
+      handleKeyPress.default(evt, rootElement, rowIndex, colIndex);
+      expect(handleKeyPress.default).to.have.been.calledOnce;
+      expect(evt.stopPropagation).to.have.been.calledOnce;
```

Comment:
AssertionError: expected spy to have been called exactly once, but it was called 0 times